src/commands/install.rs

```diff
@@ -73,11 +93,18 @@ pub fn install() -> Result<()> {
         }
     }
 
-    info!(
-        "\n\nInstalled: forc {}, fuel-core {}",
-        forc_release_latest_tag, fuel_core_release_latest_tag
-    );
-    info!("\nThe Forc toolchain is installed now. Great!");
+    info!("\nInstalled: {}\n", installed_bins_message);
+
+    if errored_bins_message.is_empty() {
+        info!("The Fuel toolchain is installed now. Great!");
+    } else if installed_bins_message.is_empty() {
+        error!("fuelup failed to install: {}", errored_bins_message)
+    } else {
+        info!(
+            "The Fuel toolchain is partially installed.\nfuelup failed to install: {}",
+            errored_bins_message
+        );
+    };
```

Comment:
Are there better ways to print these messages? Clearer instructions to check for, etc.